action.yml

```diff
@@ -2,7 +2,7 @@
 
 name: Setup MATLAB
 description: >-
-  Set up MATLAB on a Linux-based GitHub-hosted runner
+  Set up a specific version of MATLAB
```

Comment:
On line 9, I see we still say R2020a or later. Shouldn't it now be R2021a or later?

Reply:
Good catch! updated